service/history/workflow/test_util.go

```diff
@@ -37,11 +38,12 @@ import (
 func TestLocalMutableState(
 	shard shard.Context,
 	eventsCache events.Cache,
+	ns *namespace.CacheEntry,
 	logger log.Logger,
 	runID string,
 ) *MutableStateImpl {
 
-	msBuilder := NewMutableState(shard, eventsCache, logger, tests.LocalNamespaceEntry, time.Now().UTC())
+	msBuilder := NewMutableState(shard, eventsCache, logger, ns, time.Now().UTC())
```

Comment:
just curious why this change? is there case where we call this TestLocalMutableState() with cache entry other than tests.LocalNamespaceEntry?

Reply:
Yes, that's exactly it.